libraries/AP_RCProtocol/AP_RCProtocol.cpp

```diff
@@ -152,10 +152,6 @@ bool AP_RCProtocol::process_byte(uint8_t byte, uint32_t baudrate)
     uint32_t now = AP_HAL::millis();
     bool searching = (now - _last_input_ms >= 200);
 
-#ifndef IOMCU_FW
```

Comment:
I think we need to set it in both places, as if we don't have an added uart then process_byte() can still be used, and needs to honor the enabled protocols

Reply:
ok